.github/workflows/ci.yml

```diff
@@ -29,6 +29,8 @@ jobs:
 
       - name: Compile and Test
         run: |
+          mkdir config
+          echo -e "import Config\nconfig :adbc, drivers: [:sqlite]" > config/config.exs
```

Comment:
Yeah, it worked here locally. We can remove this and instead download it in test helper.exs. It is good that it tests that dynamic loading works (but then I guess we have no tests that the compilation version works). :D